projects/bootloader/scripts/jump_application.py

```diff
@@ -0,0 +1,40 @@
+"""This client script handles the jump-to-application process of controller boards."""
+
+import time
+import can
+
+from can_datagram import Datagram
+from can_datagram import DatagramSender
+from can_datagram import DatagramListener
+
+DATAGRAM_TYPE_ID = 5
+DATAGRAM_DATA = []
+
+
+def jump_to_application(node_ids):
+    """This function sends a jump-to-application datagram to specified boards,
+    receives a response, then returns the status"""
+    sender = DatagramSender(receive_own_messages=True)
+    listener = DatagramListener(trigger_callback)
+
+    can.Notifier(sender.bus, [listener])
```

Comment:
One other thing: you'll probably want to assign this to a variable to make sure it doesn't get garbage-collected.